modules/apps/powwow/powwow-portlet/docroot/WEB-INF/src/com/liferay/powwow/service/impl/PowwowMeetingOccurrenceLocalServiceImpl.java

```diff
@@ -79,6 +79,12 @@ public PowwowMeetingOccurrence addPowwowMeetingOccurrence(
 		return powwowMeetingOccurrence;
 	}
 
+	public int countByStatusAndEndTimeLE(
```

Comment:
Could you change the name to `countByStatusAndEndTimeLT`? which will reflect the meaning of `... less than`. for `countByStatusAndEndTimeLE`, the common understanding could be `... less than or equal`